python/test/unit/geometry/test_bounding_box_tree.py

```diff
@@ -496,3 +497,82 @@ def test_surface_bbtree_collision(dtype):
 
     collisions = compute_collisions_trees(bbtree1, bbtree2)
     assert len(collisions) == 1
+
+@pytest.mark.parametrize("dtype", [np.float32, np.float64])
+def test_determine_point_ownership(dtype):
+    """Find point owners (ranks and cells) using bounding box trees + global communication
+    and compare to point ownership data results."""
+    comm = MPI.COMM_WORLD
+    rank = comm.Get_rank()
+
+    tdim = 2
```

Comment:
Rather get `tdim` from `mesh.topology.dim` after mesh creation, in case we ever decide to change this to a 3D problem.

Reply:
The test also runs in 3D in the next iteration.